src/protocols/tulip.ts

```diff
@@ -23,6 +23,9 @@ import {
 } from "../types";
 import { Gateway } from "@dappio-wonderland/gateway-idls";
 import { NATIVE_SOL, TULIP_ADAPTER_PROGRAM_ID, WSOL } from "../ids";
+import { getMultipleAccounts } from "@dappio-wonderland/navigator/dist/utils";
+
+const TULIP_VAULT_ACCOUNT_LOOKUP = new anchor.web3.PublicKey("DuUSqiffUKEZwEvKBz9iDY2LZqt7LmLkPihy8E8Vfwyq");
```

Comment:
1. ~~`TULIP_VAULT_ACCOUNT_LOOKUP`~~ => `TULIP_ACCOUNT_LOOKUP_TABLE`
2. Add descriptive comment for lookup table here